package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "brainstem-js",
+  "repository": "https://github.com/mavenlink/brainstem-js",
+  "filename": "brainstem",
+  "standalone": "Brainstem",
+  "version": "0.4.0-2",
+  "author": {
+    "name": "Mavenlink",
+    "email": "dev@mavenlink.com"
+  },
+  "license": "MIT",
+  "main": "lib/brainstem.js",
+  "scripts": {
+    "prepublish": "gulp build-module",
+    "test": "gulp test"
+  },
+  "dependencies": {
+    "backbone": "^1.1.2",
+    "jquery": "^2.1.4",
+    "underscore": "^1.4.4"
```

Comment:
Personally, I prefer locking the versions of all libraries to prevent issues with dependencies mishandling semver. I'd extend that to dev dependencies as well, but bare minimum we may consider locking these versions in especially since our main app locks them in.